ansible/roles/vm_set/tasks/stop_ptf_portchannel.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This file is almost same as ptf_portchannel.yml, except the bottom task. Could you parameterize ptf_portchannel.yml to support both start (default) and stop `cmd`?

Reply:
You can refer to https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/vm_set/tasks/remove_topo.yml#L20
The idea is to pass in a variable with name like `ptf_portchannel_action` while calling playbook `ptf_portchannel.yml`.
Value of the variable can be `start` during add-topo and `stop` during remove-topo. Then update the `ptf_portchannel.yml` book to act differently according to value of the variable.

Reply:
Updated.